resources/views/filament/alerts/alert-banner.blade.php

```diff
@@ -0,0 +1,43 @@
+@props(['alertBanner'])
+
+@php
+    if ($alertBanner) {
```

Comment:
The if statement can be removed and this `@php` block can be moved under the `@isset` below.